python-sdk/src/configuration.rs

```diff
@@ -0,0 +1,70 @@
+// We export configuration object into Python world with a limited set of operations to allow
+// backend to pass on configuration when initializing the frontend.
+use std::{borrow::Cow, sync::Arc};
+
+use pyo3::{
+    exceptions::{PyRuntimeError, PyValueError},
+    prelude::*,
+    types::PySet,
+};
+
+use eppo_core::Configuration as CoreConfiguration;
+
+/// Eppo configuration of the client, including flags and bandits configuration.
+///
+/// Internally, this is a thin wrapper around Rust-owned configuration object.
+#[pyclass(frozen, module = "eppo_client")]
+pub struct Configuration {
+    pub configuration: Arc<CoreConfiguration>,
+}
+
+#[pymethods]
+impl Configuration {
+    #[new]
+    #[pyo3(signature = (*, flags_configuration))]
+    fn py_new(flags_configuration: &[u8]) -> PyResult<Configuration> {
+        let flag_config = serde_json::from_slice(flags_configuration).map_err(|err| {
+            PyValueError::new_err(format!("argument 'flags_configuration': {err:?}"))
+        })?;
+        Ok(Configuration {
+            configuration: Arc::new(CoreConfiguration::from_server_response(flag_config, None)),
+        })
+    }
+
+    // Returns a set of all flag keys that have been initialized.
+    // This can be useful to debug the initialization process.
+    fn get_flag_keys<'py>(&'py self, py: Python<'py>) -> PyResult<Bound<PySet>> {
+        PySet::new_bound(py, self.configuration.flags.flags.keys())
+    }
+
+    // Returns a set of all bandit keys that have been initialized.
+    // This can be useful to debug the initialization process.
+    fn get_bandit_keys<'py>(&'py self, py: Python<'py>) -> PyResult<Bound<PySet>> {
+        PySet::new_bound(
+            py,
+            self.configuration
+                .bandits
```

Comment:
minor but we're deprecating the array-valued `bandits` in favor of object-valued `banditReferences`. Both have a top-level object though whose keys are the bandit keys, so should be an easy swap.